eox_core/edxapp_wrapper/backends/users_h_v1.py

```diff
@@ -16,7 +16,12 @@
     LANGUAGE_KEY
 )
 from openedx.core.djangoapps.user_api.accounts.api import (  # pylint: disable=import-error
-    check_account_exists
+    check_account_exists,
+    update_account_settings
```

Comment:
Why?

Reply:
Leaving a trailing comma guarantees next time we edit the code it will be a change for one line of the code. Note that before your change we had `check_account_exists,` as the last imported member. @anmrdz

Reply:
Ok, I've made the changes.